Lesson 3 HomeWork.playground/Contents.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+// Функции умножения
+func multiplication(firstMultiplier: Double, secondMultiplier: Double) -> Double {
+    return firstMultiplier * secondMultiplier
+}
+func multiplication(firstMultiplier: Int, secondMultiplier: Int) -> Int {
+    return firstMultiplier * secondMultiplier
+}
+
+print(multiplication(firstMultiplier: 2, secondMultiplier: 4))
+print(multiplication(firstMultiplier: 4.5, secondMultiplier: 0.5))
+
+// функции деления
+func division(divident: Double, divider: Double) -> Double {
+    return divident / divider
+}
+func division(divident: Int, divider: Int) -> Int {
+    return divident / divider
+}
+
+print(division(divident: 12, divider: 12))
+print(division(divident: 24.8, divider: 12.4))
+ 
+// функции сложения
+func addition(_ firstTerm: Int, _ secondTerm: Int) -> Int{
+    return firstTerm + secondTerm
+}
+func addition(_ firstTerm: Float, _ secondTerm: Float) -> Float{
+    return firstTerm + secondTerm
+}
+
+print(addition(1,2))
+print(addition(2.2,2))
+
+// функции вычитания
+func substraction(_ minuend: Int, _ subtrahend: Int) -> Int{
+    return minuend - subtrahend
+}
+func substraction(_ minuend: Double, _ subtrahend: Double) -> Double{
+    return minuend - subtrahend
+}
```

Comment:
It's better to use verbs if we can in function names. For example, `add` instead of `addition` and so on